modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/helper/ValidationHelper.java

```diff
@@ -31,6 +32,19 @@
 @Component(service = ValidationHelper.class)
 public class ValidationHelper {
 
+	public boolean isValidAPIApplicationCompanyId(long objectEntryId) {
+		ObjectEntry objectEntryAPIApplication =
```

Comment:
What happen if the `objectEntryAPIApplication` is null? Should we return a new validation message?

Reply:
Hi @sergiojimcos , as there is already a verification for api application value [on this line](https://github.com/liferay-headless/liferay-portal/blob/a62284f6738adadbceb9f8f409fc99dc73ad10ee/modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APISchemaRelevantObjectEntryModelListener.java#L61), I think we can just return false. New commit: https://github.com/liferay-headless/liferay-portal/pull/1896/commits/892d2e6029ccad4f218d82337598bbb5c00c66ef